src/components/AddButton.js

```diff
@@ -0,0 +1,53 @@
+import React,  { Component, } from "react";
+import PropTypes from "prop-types";
+import { Button, View, TouchableOpacity, Text, Image} from 'react-native';
+import styled from 'styled-components';
+import AddIcon from '../assets/images/icons/add-icon.png';
+
+class AddButton extends Component {
+    constructor(props) {
+      super(props);
+    }
+  
+    render() {
+      return (
+      <AddCustomButton type="submit" >
+          <Image style={{ backgroundColor: 'transparent', tintColor: '#5089E9', bottom: 10, height: 40, width: 40 }} source= {AddIcon}/>
```

Comment:
Use styled-components for the styling here (avoiding using two style methods will make the code more readable and clean). Similar example in App.js with the TabIcon tag